crates/wasmtime/Cargo.toml

```diff
@@ -302,6 +301,7 @@ std = [
   'wasmtime-environ/std',
   'object/std',
   'once_cell',
+  'wasmtime-fiber/std',
```

Comment:
I might recommend `wasmtime-fiber?/std` (note the `?`) to only enable the `wasmtime-fiber` dependency if it's already activated via some other means.

Reply:
Done.